src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -6,8 +6,8 @@
     "System.Net.Http": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-23921",
-    "System.ServiceModel.Duplex": "4.0.1-rc3-23921",
-    "System.ServiceModel.Http": "4.0.11-rc3-23921",
+    "System.ServiceModel.Duplex": "4.1.0-rc3-23921",
```

Comment:
I don't think  Duplex needed to be updated, we don't have a 4.1 version for it.
After fixing the Duplex versions I get an issue with System.Linq.Expressions, looks like something is downgrading it to 4.0.0, working on it.